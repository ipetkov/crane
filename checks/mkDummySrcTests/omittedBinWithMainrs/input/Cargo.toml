[package]
name = "mybin" # Loadbearing name matching bin
version = "0.1.0"
edition = "2021"
