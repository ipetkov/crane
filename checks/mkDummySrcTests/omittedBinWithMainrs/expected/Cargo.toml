[package]
build = "cranespecific-dummy.rs"
edition = "2021"
name = "mybin"
version = "0.1.0"
